pom.xml

```diff
@@ -29,7 +29,7 @@
         <kotest.version>5.9.1</kotest.version>
         <token-validation-spring.version>5.0.5</token-validation-spring.version>
         <okhttp3.version>4.9.1</okhttp3.version><!-- Inntil videre nødvendig for token-validation-spring-test-->
-        <openhtmltopdf.version>pdfbox2-65c2c5010f84b2daa5821971c9c68cd330463830</openhtmltopdf.version>
+        <openhtmltopdf.version>1.1.21</openhtmltopdf.version>
```

Comment:
Jeg skjønner jo lite av dette, men hvordan påvirker det dette problemet som Tor snakket om i https://github.com/navikt/familie-tilbake/pull/1378 ?

Reply:
`also { it.isEnableGsub = false }` løser egentlig det problemet der.

Reply:
Testet i preprod - ft, fi og f* ser ut til å fungere som forventet nå.